Nextjs documentation update (#239)
tigrisdata/tigris-docs: docs/guides/nextjs/index.mdx

### Initialize your Next.js app

Next.js allows you to bootstrap apps using [create-next-app](https://nextjs.org/docs/api-reference/create-next-app)
utility:

```shell
npx create-next-app --example with-tigris
```

This will prompt for the project name. Enter `my-tigris-app` and hit the Enter key. 

<details>
  <summary>Output</summary>
  <TerminalWindow>
    <CodeBlock language="shell">{createNextAppOutput}</CodeBlock>
  </TerminalWindow>
</details>

Finally, navigate into the project directory and run the application:

```
cd my-tigris-app
npm run dev
```

### Add the Tigris cloud credentials
To connect your application with Tigris, you need to generate 

> Your applicationrequires credentials to be able to use Tigris. Let's generate the credentials and

Reviewer: ```suggestion
Your application requires credentials to be able to use Tigris. Let's generate the credentials and
```

add them to the environment file.

```shell
tigris create application starter-todo-app "Starter todo application"
```

> The this will create the credentials and print it on the terminal screen:

Reviewer: ```suggestion
Credentials will be created and printed on the terminal screen:
```
